chore: update teraform provider docs (#665)
bytebase/bytebase.com: content/docs/get-started/work-with-terraform/overview.md

1. Install [Terraform](https://developer.hashicorp.com/terraform/downloads?product_intent=terraform)

2. Start Bytebase to get the server running.

## Configuration

### Create service account

Redirect to Bytebase member management page (/setting/member).

Give a specific name to identifier your service account, and grant the **Owner** or **DBA** role to it.

![create-service-account](/docs/get-started/work-with-terraform/create-service-account.webp)

> After creation, you can copy the service key as `service_key` and the email as `service_account` to initialize the Bytbase provider in next step.

Reviewer: Just double check the copy button only appears the first time? Since users can't get the password and copy afterwards.

Author: Right, nice catch. I add a warning message in the doc to remind this
<img width="991" alt="图片" src="https://user-images.githubusercontent.com/10706318/204699899-393ed773-8e67-4358-aa50-7b4d9a7b9ca2.png">

> 1. Install [Terraform](https://developer.hashicorp.com/terraform/downloads?product_intent=terraform)

Reviewer 2: '.' missing

2. Start Bytebase to get the server running.

## Configuration

### Create service account

> Redirect to Bytebase member management page (/setting/member).

Reviewer 2: Click **Settings** on the navigation bar, and then click **Workspace** > **Members**.

Name your service account, and grant the **Owner** or **DBA** role to it.

![create-service-account](/docs/get-started/work-with-terraform/create-service-account.webp)

After creation, you can copy the service key as `service_key` and the email as `service_account` to initialize the Bytbase provider in next step.

<hint-block type="warning">

> Please copy the service key once you have created the service account. The key will disappear if you refresh the page.

Reviewer: ```suggestion
You can only copy the key right after creating the service account. The key will disappear if you refresh the page.
```
